.github/workflows/ci.yml

```diff
@@ -1,4 +1,5 @@
-on:
+---
+on:  # yamllint disable-line rule:truthy
```

Comment:
i have it set up locally for all yaml files and it started screaming in my face.

but sure, it can be removed, and i just did that. "on" is non-standard as boolean value, but this is a dictionary _key_ which are always strings in the json/yaml data model, so that yamllint warning is a bit strict indeed.